### PR TITLE
Gives candles a new animation

### PR DESCRIPTION
**Pull request by Greniza**
## About The Pull Request

Gives burning candles and altar candles a more fluid and good-looking animation.

## Why It's Good For The Game

The old candle animation is flat and boring.
old-candle-big

My new animation is more bouncy and more interesting. It also has a little puff of smoke.
new-candle-big

I also added altar animations for the altar.

Old (flat, boring, and inconsistent with candle artstyle)
old-altar-big

New (Bouncy, exciting, artstyle-conforming, and uses overlays!)
new-altar-big

## Changelog
:cl:
imageadd: New candle animations
imageadd: New animations for altar candles
/:cl:

This isn't really a 3 GBP PR, I only really expect one from this.

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
> This isn't really a 3 GBP PR, I only really expect one from this.

you get what you're owed, and sprites are worth 3

**Comment by tralezab** [comment]
dawww

**Comment by Greniza** [comment]
> > This isn't really a 3 GBP PR, I only really expect one from this.
> 
> you get what you're owed, and sprites are worth 3

Just trying to avoid being accused of farming GBP

**Comment by LemonInTheDark** [comment]
> 
> 
> > > This isn't really a 3 GBP PR, I only really expect one from this.
> > 
> > 
> > you get what you're owed, and sprites are worth 3
> 
> Just trying to avoid being accused of farming GBP

farming gbp is good, it means you're making the game better. no need to worry about it

**Review by Twaticus — APPROVED** [review]

File: code/modules/religion/religion_structures.dm [hunk 1]
```diff
@@ -43,6 +49,7 @@
 			icon = sect_to_altar.altar_icon
 		if(sect_to_altar.altar_icon_state)
 			icon_state = sect_to_altar.altar_icon_state
+	update_overlays() //Light the candles!
```
Comment (ninjanomnom) [inline code/modules/religion/religion_structures.dm:52]: This just returns a list and does nothing to update the overlays

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/religion/religion_structures.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/modules/religion/religion_structures.dm:52]: You want `update_appearance` here

Comment (Greniza) [reply code/modules/religion/religion_structures.dm:52]: I stored the candle flame for the non-standard altars as an overlay to save on space and time, you're sure?

Comment (Greniza) [reply code/modules/religion/religion_structures.dm]: i'm a big dumb baby that doesn't know how this code works lmao

**Pull request merged**